src/txs/Tx.tsx

```diff
@@ -11,8 +11,7 @@ import { head, isNil } from "ramda"
 import AccountBalanceWalletIcon from "@mui/icons-material/AccountBalanceWallet"
 import ErrorOutlineIcon from "@mui/icons-material/ErrorOutline"
 import { isDenom, isDenomIBC, readDenom } from "@terra.kitchen/utils"
-import { Coin, Coins, LCDClient } from "@terra-money/terra.js"
-import { CreateTxOptions, Fee } from "@terra-money/terra.js"
+import { Coin, Coins, LCDClient, CreateTxOptions, Fee } from "@terra-money/terra.js"
```

Comment:
Why?

Reply:
> Why?

why what?

Reply:
Why is it changed?

Reply:
We had added an import from terra.js than we then simply removed.
It is confusing IMHO to have two separate import lines from the same module, but we can revert if you like, no worries.